app/assets/javascripts/hello-dev.js

```diff
@@ -18,5 +18,5 @@ var bigDev =
                                                      \n';
 console.log(
   bigDev,
-  "Hey there! Interested in the code behind dev.to? Well you're in luck - we're open source! Come say hi, tell us what you're debugging, or even lend a hand in our repo - https://github.com/thepracticaldev/dev.to/"
+  "Hey there! Interested in the code behind dev.to? Well you're in luck - we're open source! Come say hi, tell us what you're debugging, or even lend a hand in our repo - https://github.com/thepracticaldev/dev.to/ \n \Did you find a bug or vulnerability? Check out our bug bounty info here: https://dev.to/security"
```

Comment:
It'd be great if you could change the language/copy here, too, for clarity :) For example:
> Did you find a vulnerability? Check out our security page here: https://dev.to/security